### ISSUE TITLE
Formatting of java.time.LocalTime does not work

### ISSUE THREAD

**Issue by cplaetzinger** [thymeleaf/thymeleaf-extras-java8time#2]
I'm trying to format an instance of type java.time.LocalTime

```
${#temporals.format(localTime)}
```

This leads to the following exception:

```
Caused by: java.lang.IllegalArgumentException: Cannot format object of class "java.time.LocalTime" as a date
    at org.thymeleaf.extras.java8time.util.TemporalFormattingUtils.zonedTime(TemporalFormattingUtils.java:183)
    at org.thymeleaf.extras.java8time.util.TemporalFormattingUtils.formatDate(TemporalFormattingUtils.java:161)
```

**Comment by jmiguelsamper** [comment]
Hi Christian,

I'm afraid you're right. We will publish a fix ASAP.

Thanks

**Comment by cplaetzinger** [comment]
Thanks for your fast reply. Was not sure if I use it in a proper way.

**Comment by jmiguelsamper** [comment]
As a temporary workaround, it will work if you supply a pattern as a second argument.

**Comment by jmiguelsamper** [comment]
This is fixed in 2.1.1-SNAPSHOT (see http://www.thymeleaf.org/faq.html#use-snapshot) but please be aware of issue #3
